base/cvd/cuttlefish/host/commands/cvd/fetch/fetch_tracer.cpp

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright (C) 2024 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "host/commands/cvd/fetch/fetch_tracer.h"
+
+#include <chrono>
+#include <iomanip>
+#include <memory>
+#include <mutex>
+#include <sstream>
+#include <vector>
+
+namespace cuttlefish {
+namespace {
+
+struct Phase {
+  std::string name;
+  std::chrono::seconds duration;
+  std::optional<size_t> size;
```

Comment:
let's add the unit name to the name, i.e `size_bytes`?

Reply:
done